code/game/machinery/kitchen/microwave.dm

```diff
@@ -449,6 +464,14 @@
 	if(emagged || arcanetampered)
 		max_temperature = INFINITY //If it's been messed with, let it heat more than that.
 	cooked.reagents.heating(thermal_energy_transfer, max_temperature)
+	var/cook_temp = COOKTEMP_READY//100°C
+	if(emagged || arcanetampered)
+		cook_temp = COOKTEMP_EMAGGED//8.000.000°C
```

Comment:
1400x hotter than the surface of the sun?
I know it's for silly memes but this might be overkill

Reply:
It's actually a funny little reference to [an old Looney Toons episode](https://www.youtube.com/watch?v=5pEvmw_T6-U).

I was myself unsure if this might cause some issues but having actually tested in-game, the burn damage from heating a donk pocket coming out straight of the oven at this temperature, while melting my face, didn't even put me into crit and healed over time, which I assume is due to some damage capping on scalding reagent splashing implemented in the past.

And I have another reason to keep it at that temperature, with my thermal entropy subsystem (which causes things to reach room temperature much faster than thermal dissipation), it actually only takes about 5 minutes for food at this temperature to reach room temperature.

Reply:
Wouldn't the "emagged always makes burned messes and plays the dramatic hell's kitchen sound effect" interaction go first, thereby only ever giving you hyper heated burned messes?

Reply:
How does it work now? Is it ONLY the 8 million degrees food? Or do you still get the hell's kitchen mess?

Reply:
You'll still get the hell's kitchen soundbyte if you mess up your recipe. If the recipe is good or you're just re-heating something, it will be 8 million degrees, and on fire :)

Reply:
Alright that works. I was kinda half-expecting a chance for either of them to be applied, on emag, so the microwave either burns or overheats.
This seems better, as the previous emag interaction was only a nuisance if you had a screwdriver and crowbar.
Do the foodstuffs burn from being at that temperature or is it just "steaming hot" and edible (but actually 8M degrees)

Reply:
one thing is that there is a global scaling factor applied to all thermal dissipation that can be tweaked as desired.

Reply:
Yes, I have something similar in my own system.